### PR TITLE
libgnt: Add package

### PR DESCRIPTION
**Pull request by ghost**
Add the GLib Ncurses Tool to the repository.
It's a dependency of the current Pidgin version.
Needs #7721 merged first to be built successfully.

### CODE REVIEW

File: components/library/libgnt/Makefile [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#
+
+BUILD_BITS=64
+BUILD_STYLE=meson
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		libgnt
+COMPONENT_VERSION=	2.14.3
+COMPONENT_REVISION=	1
+COMPONENT_PROJECT_URL=	https://www.pidgin.im
```
Comment (AndWac) [inline components/library/libgnt/Makefile]: Please remove the COMPONENT_REVISION as this is a new package.

File: components/library/libgnt/Makefile [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+COMPONENT_SUMMARY=	The GLib Ncurses Toolkit
+COMPONENT_FMRI=		library/libgnt
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:57f5457f72999d0bb1a139a37f2746ec1b5a02c094f2710a339d8bcea4236123
+COMPONENT_ARCHIVE_URL=	http://sourceforge.net/projects/pidgin/files/$(COMPONENT_NAME)/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)/download
+COMPONENT_CLASSIFICATION=	System/Libraries
```
Comment (AndWac) [inline components/library/libgnt/Makefile:30]: This should use https.

File: components/library/libgnt/Makefile (hunk 1 above)
Comment (AndWac) [inline components/library/libgnt/Makefile]: I cannot fint libgnt on www.pidgin.im but on https://keep.imfreedom.org/libgnt/libgnt/

Comment (jgmills) [reply components/library/libgnt/Makefile]: It's also unlikely that I would have applied a copyright to the file in 2017.

**Comment by ghost** [comment]
It's all fixed now.
Sorry for the wrong copyrights,I copied the Makefile from the Pidgin package and forgot to change it.

**Pull request merged**